### ISSUE TITLE
Compute value & uncertainty from GSAS-II parameters & covariance matrix

### ISSUE THREAD

**Issue by briantoby** [AdvancedPhotonSource/GSAS-II#18]
Provide users with a mechanism to supply an arbitrary equation into GSAS-II that derives a value from the fitted parameters. The derivatives for that equation can then be determined numerically and the uncertainty for that parameter can then be evaluated.

**Comment by briantoby** [comment]
started on this. 

Mathew suggests looking at Automatic differentiation (https://jax.readthedocs.io/en/latest/automatic-differentiation.html). See https://jax.readthedocs.io/en/latest/automatic-differentiation.html#taking-gradients-with-jax-grad.
